apps/communications/contacts/js/contacts.js

```diff
@@ -276,28 +270,22 @@ var Contacts = (function() {
     HeaderUI.hideAddButton();
     contactsList.clearClickHandlers();
     contactsList.handleClick(function addToContactHandler(id) {
-      var data = {};
+      window.location.href =
+        '/contacts/views/form/form.html?action=update&contact=' + id;
+      
+      var optionalParams = '';
+
       if (params.hasOwnProperty('tel')) {
-        var phoneNumber = params.tel;
-        data.tel = [{
-          'value': phoneNumber,
-          'carrier': null,
-          'type': [TAG_OPTIONS['phone-type'][0].type]
-        }];
+        optionalParams= '&isActivity=true&tel=' + params.tel;
```

Comment:
And we could handle the optional params in that method:

`getContatDetailUrl(id, extraParams)`